docs/data/stashcache/run-stashcache-container.md

```diff
@@ -0,0 +1,195 @@
+      when usage goes above the high watermark, the cache deletes until it hits the low watermark
+- `XC_PORT`: TCP port that XCache listens on
+- `XC_RAMSIZE`: Amount of memory to use for storing blocks before writting them to disk. (Use higher for slower disks).
+- `XC_BLOCKSIZE`: The size of the blocks in the cache
+- `XC_PREFETCH`: Number of blocks to prefetch from a file at once. 
+       This is a value of how aggressive the cache is to request portions of a file. Set to `0` to disable
+
+### Optimizing Stashcache ###
+
+#### Network Optimization ####
+
+For caches that are connected to NIC's over `40Gbps` we recommend to disable the virtualized network and "bind" the container to the host network:
+
+```console
+user@host $ docker run --rm  \
+             --network="host" \
+             --volume <HOST PARTITION>:/cache \
+             --env-file=/opt/xcache/.env \
+             opensciencegrid/stash-cache:stable
+```
+
+#### Memory Optimization ####
+
+Stashcache uses the hosts memory in two ways:
+
+1. Uses the own linux kernel as a way to cache files read from disk
+1. As a buffer for writting blocks of files first in memory and then in disk (to account for slow disks).
+
+An easy way to increase the performance of stashcache is to assign it more memory. You can use the [docker option](https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory) `--memory` and set it up to at least twice of `XC_RAMSIZE`.
+
+```console
+user@host $ docker run --rm --publish <HOST PORT>:8000 \
+             --memory=64g \
+             --volume <HOST PARTITION>:/cache \
+             --env-file=/opt/xcache/.env \
+             opensciencegrid/stash-cache:stable
+```
+
+
+#### Multiple Disks ####
+
+For caches that store over `10 TB` or that have assigned space for storing the cached files over multiple partitions (`/partition1, /partition2, ...`) we recommend the following.
+
+1. Create a config file `90-my-stash-cache-disks.cfg` with the following contents
+
+        :::file
+        pfc.spaces data
+        oss.space data /data1
+        oss.space data /data2
+        .
+        .
+        .
+
+1. Run the container with the following options:
+
+        :::console
+        user@host $ docker run --rm --publish <HOST PORT>:8000 \
+             --volume <HOST PARTITION>:/cache \
+             --volume /partition1:/data1 \
+             --volume /partition2:/data2 \
+             --volume /opt/xcache/90-my-stash-cache-disks.cfg:/etc/xrootd/config.d/90-stash-cache-disks.cfg \
+             --env-file=/opt/xcache/.env \
+             opensciencegrid/stash-cache:stable
+
+!!! note
+    Under this configuration the `<HOST PARTITION>` is not used to store the files rather to store symlinks to the files in `/partition1` and `/partition2`
+
+!!! warning
+    For over 100 TB of assigned space we highly encourage to use this setup and mount `<HOST PARTITION>` in solid state disks or NVME.
```

Comment:
Let's move this sentence to the intro paragraph of this section (and get rid of the warning)